### PR TITLE
[WFLY-9637] Stop using deprecated OFE constructor

### PR DESCRIPTION
**Pull request by bstansberry**
https://issues.jboss.org/browse/WFLY-9637

### CODE REVIEW

File: testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/timerservice/mgmt/AbstractTimerManagementTestCase.java [hunk 1]
```diff
@@ -277,4 +262,12 @@ protected void assertNoTimers() {
     protected String getCalendarTimerDetail(){
         return this.bean.getComparableTimerDetail();
     }
+
+    private ModelNode executeForResult(ModelNode operation) throws OperationFailedException, IOException {
+        final ModelNode outcome = this.managementClient.getControllerClient().execute(operation);
+        if (!Operations.isSuccessfulOutcome(outcome)) {
+            throw new OperationFailedException("Failed executing " + operation.toString());
```
Comment (jmesnil) [inline testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/timerservice/mgmt/AbstractTimerManagementTestCase.java]: this message should be i18ned.

**Comment by jmesnil** [comment]
failures looks related to the commit.

**Comment by stuartwdouglas** [comment]
Retest this please

**Comment by stuartwdouglas** [comment]
The the change to triggerTimer had left in the original execute call so the timer was triggered twice. I have deleted the line and assuming CI now passes this should be good to merge.

**Review by stuartwdouglas — APPROVED** [review]

**Pull request merged**